external-crates/move/move-compiler/src/typing/translate.rs

```diff
@@ -76,7 +80,7 @@ fn module(
     let functions = nfunctions.map(|name, f| function(context, name, f, false));
     assert!(context.constraints.is_empty());
     context.env.pop_warning_filter_scope();
```

Comment:
If you move this to line 95 (after calling `gen_unused_warnings`) I don't think we need the warning scope addition inside of `gen_unused_warnings`? (unless I'm missing something)

Reply:
Aha! I am missing something! Doh! You also include the filter of the function/struct with the warning. I get it now :) Sorry...